src/components/card/index.js

```diff
@@ -205,16 +212,20 @@ class Card extends PureBaseComponent {
     );
   }
 
-  renderChildren() {
+  renderChildren = () => {
     const children = React.Children.map(this.props.children, (child, index) => {
```

Comment:
instead of keeping indices of null items, maybe we can just filter those children and ignore them. 
and then the `calcChildPosition` can be simpler..

Reply:
Done